daemon/manager/manager.go

```diff
@@ -67,6 +68,7 @@ func NewCommand() *cobra.Command {
 	cmd.AddCommand(version.NewCmd())
 	cmd.AddCommand(dump.NewCmd())
 	cmd.AddCommand(logcmd.NewCmd())
+	cmd.AddCommand(uninstall.NewUninstallCmd())
```

Comment:
```suggestion
	cmd.AddCommand(uninstall.NewCmd())
```

Be consistent with other subcommands and the pkg name already explains the purpose.